ga4gh/datarepo.py

```diff
@@ -110,15 +110,17 @@ def getNumReferenceSets(self):
         """
         return len(self._referenceSetIds)
 
-    def getOntology(self, name):
+    def getOntologyTermMapByName(self, name):
         """
-        Returns an ontology map by name
+        Returns an ontologyTermMap by name
         """
-        return self._ontologyNameMap.get(name, None)
+        if name not in self._ontologyNameMap:
+            raise exceptions.OntologyNameNotFoundException(name)
```

Comment:
In the variant annotation files I was working with, the transcript effect terms did not always map directly to the SO terms. Sometimes synonyms are used, or the developer adds terms that are not present. For that reason, we'll want to be sure this exception gets handled and doesn't throw to the user. When working directly with VCF there are not enough guarantees that the terms will map.

Forcing users to specify a specific SO version when adding a variant annotation set could help mitigate this, however, it is doubtful they will know unless they are the creators of the tool used to generate the annotations.